### DISCUSSION TITLE
edk2 submodule UnitTestFrameworkPkg/Library/SubhookLib/subhook (github.com/Zeex/subhook.git) not available

### DISCUSSION THREAD

**Discussion by totoroliu** [tianocore/edk2#6401] [Q&A] [answered]
Since Nov.3, 2024,
https://github.com/Zeex repositories (including subhook.git) becomes private.

![image](https://github.com/user-attachments/assets/6ebbcace-99ef-4c06-9d33-3d5525a60605)

Does anyone have subhook.git available somewhere?

**Comment by mdkinney** [comment, marked as answer]
Closing as duplicate discussion https://github.com/tianocore/edk2/issues/6398
